src/redux/reducers/StudentReducer.js

```diff
@@ -0,0 +1,21 @@
+import * as types from '../actions/ActionTypes';
+
+export default function (state = {}, action) {
+
+  const { payload } = action;
+
+  switch (action.type) {
+    case types.FETCH_STUDENTS_FULFILLED:
+      return {
+        students: action.payload
```

Comment:
I don't know if it's intended to return a state with just `students` (other field in the state would be lost)